flow/libs.js

```diff
@@ -38,10 +38,10 @@ type Attrs = $ReadOnly<{[?string]: any}>
 
 declare interface Mithril {
 	// We would like to write a definition which allows omitting Attrs if all keys are optional
-	(component: string | Component | MComponent<void> | Class<MComponent<void>>, children?: Children): Vnode<any>;
+	(component: string | MComponent<any> | Class<MComponent<any>>, children?: Children): Vnode<any>;
```

Comment:
You replaced `void` with any `here`. The idea was that you cannot render a component without attrs, if it needs any